frontend/src/components/common/episode/episode.tsx

```diff
@@ -0,0 +1,50 @@
+import { useAppSelector, useDispatch, useLocation, useEffect } from 'hooks/hooks';
```

Comment:
It looks like this should be considered as a page, so let's move this to the root level of the components directory